.travis.yml

```diff
@@ -9,17 +9,17 @@ env:
   - MACOS_FRAMEWORK_SCHEME="Willow macOS"
   - TVOS_FRAMEWORK_SCHEME="Willow tvOS"
   - WATCHOS_FRAMEWORK_SCHEME="Willow watchOS"
-  - IOS_SDK=iphonesimulator10.3
-  - OSX_SDK=macosx10.12
-  - TVOS_SDK=appletvsimulator10.2
-  - WATCHOS_SDK=watchsimulator3.2
+  - IOS_SDK=iphonesimulator11.0
+  - OSX_SDK=macosx10.13
+  - TVOS_SDK=appletvsimulator11.0
+  - WATCHOS_SDK=watchsimulator4.0
   - EXAMPLE_SCHEME="iOS Example"
   matrix:
-    - DESTINATION="OS=10.2,name=iPhone 6S"                  SCHEME="$IOS_FRAMEWORK_SCHEME"     SDK="$IOS_SDK"     RUN_TESTS="YES" BUILD_EXAMPLE="YES" POD_LINT="YES"
-    - DESTINATION="OS=10.2,name=iPhone 5"                   SCHEME="$IOS_FRAMEWORK_SCHEME"     SDK="$IOS_SDK"     RUN_TESTS="YES" BUILD_EXAMPLE="YES" POD_LINT="NO"
+    - DESTINATION="OS=11.0,name=iPhone 7"                   SCHEME="$IOS_FRAMEWORK_SCHEME"     SDK="$IOS_SDK"     RUN_TESTS="YES" BUILD_EXAMPLE="YES" POD_LINT="YES"
+    - DESTINATION="OS=11.0,name=iPhone 5S"                  SCHEME="$IOS_FRAMEWORK_SCHEME"     SDK="$IOS_SDK"     RUN_TESTS="YES" BUILD_EXAMPLE="YES" POD_LINT="NO"
```

Comment:
Hmmmm...we should either run tests against the iOS `10.2` or `10.3` simulator instead for the 5S. Then we're getting more coverage for the other platforms.